### PR TITLE
Readme: explain how to debug (fixes #1112)

### LINKED ISSUE #1112: Document how to debug with Node.js debugger/Chrome DevTools/etc.
Until we have a seamless debugging experience (perhaps see #929), it would be great to have instructions on how to use current debuggers like Chrome DevTools and the Node.js debugger with Ava.

The only place I've seen such instructions is where they were given to me [in an issue comment](https://github.com/avajs/ava/pull/874#issuecomment-261063096).

Would a pull-request for this be welcomed?

### PR DESCRIPTION
**Pull request by mightyiam**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -169,6 +170,22 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+## Debugging
+
+The AVA CLI runs tests in child processes.
+Thus to debug one must bypass it:
```
Comment (sindresorhus) [inline readme.md]: > AVA runs tests in child processes, so to debug tests, you need to do this workaround:

File: readme.md [hunk 2]
```diff
@@ -169,6 +170,22 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
...
+
+``` console
+$ node --inspect node_modules/ava/profile.js some/test/file.js
+```
+
+In the future this bypass may not be required;
+See #874.
```
Comment (sindresorhus) [inline readme.md]: Put it on the previous line.

File: readme.md [hunk 3]
```diff
@@ -169,6 +170,22 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
 
 When using `npm test`, you can pass positional arguments directly `npm test test2.js`, but flags needs to be passed like `npm test -- --verbose`.
 
+## Debugging
+
+The AVA CLI runs tests in child processes.
+Thus to debug one must bypass it:
+
+``` console
```
Comment (sindresorhus) [inline readme.md]: Remove space

File: readme.md [hunk 4]
```diff
@@ -169,6 +170,22 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
...
+$ node --inspect node_modules/ava/profile.js some/test/file.js
+```
+
+In the future this bypass may not be required;
+See #874.
+
+### Debugger-Specific Tips
```
Comment (sindresorhus) [inline readme.md]: Don't titleize

File: readme.md [hunk 5]
```diff
@@ -169,6 +170,22 @@ Directories are recursed, with all `*.js` files being treated as test files. Dir
...
+
+* [WebStorm](docs/recipes/debugging-with-webstorm.md)
```
Comment (sindresorhus) [inline readme.md]: `*` => `-`

**Comment by mightyiam** [comment]
The changes requested by @sindresorhus were committed.

**Pull request merged**